testbed_example/services/example_simulator/example_simulator.py

```diff
@@ -12,7 +12,7 @@ def __init__(self):
         super().__init__('example_simulator')
 
     def open(self):
-        self.model = ExampleOpticalModel()
+        self.model = ExampleOpticalModel(self.testbed.config)
```

Comment:
Nice catch on this - I also think it needs to be `self.testbed.config['simulator']`? Otherwise I am not sure what pool of configurations it tries to read, between all the different configuration files the example has. It might be automatically choose the right one, but it's better to be explicit.

Reply:
I made it go on the simulator file in the optical model, but I can defenitly change that you're right.